README.md

```diff
@@ -460,7 +460,8 @@ For example, you can write a `create_example_db.sql` SQL script that creates a t
 ```yaml
 - name: Set facts to create a test DB on primary as a pre task
   set_fact:
-    mssql_pre_input_sql_file: create_example_db.sql
+    mssql_pre_input_sql_file:
```

Comment:
```suggestion
    mssql_pre_input_sql_files:
```

Reply:
I decided that renaming mssql_pre_input_sql_file to mssql_pre_input_sql_files is an overkill, I'd rather allow mssql_pre_input_sql_file to get both lists and strings